src/ranged.cpp

```diff
@@ -1862,13 +1862,16 @@ item::sound_data item::gun_noise( bool const burst ) const
 
     noise = std::max( noise, 0 );
 
+    // Grenade launchers
     if( ammo_type() == ammotype( "40mm" ) ) {
         return { 8, _( "Thunk!" ) };
 
-    } else if( typeId() == "hk_g80") {
+    // Railguns
+    } else if( ammotype() == "12mm" || ammotype( "metal_rail" ) ) {
```

Comment:
[All broke and junk](http://i.imgur.com/0CSIoEB.png), needs to be

`    } else if( ammo_type() == ammotype( "12mm" ) || ammo_type() == ammotype( "metal_rail" ) ) {`

Did this locally, got [the correct behavior](http://i.imgur.com/0Z51MrW.png).